Scripts/Game/UserInterface/UserInterfaceWindow.cs

```diff
@@ -22,6 +22,7 @@ public interface IUserInterfaceWindow
     {
         UserInterfaceWindow Value { get; }
         bool Enabled { get; set; }
+        bool PauseWhileOpen { get; set; }
```

Comment:
Please revert this change. It is not desired for all windows to pause game. It is desirable to do this on a per-window basis as designed.